content/en/events/kcsna2021/location.md

```diff
@@ -4,15 +4,45 @@ type: docs
 weight: 5
 ---
 
-<img align="right" src="/events/kcsna2021/hotel.jpeg" width="25%">
+
+### Marriott 
+<img align="right" src="/events/kcsna2021/hotel.jpeg" width="15%">
 
 The North America Summit will take place October 11th, in Los Angeles, California
 alongside <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" rel="noopener noreferrer" target="_blank">KubeCon</a>
-at the <a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">JW Marriott LA Live</a>.
+at the <a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">JW Marriott LA Live</a>
+in Ballroom 6. There will be signage in the Marriott Lobby to point you in the
+right direction.
 
 **Address**<br>
 JW Marriott Los Angeles L.A. LIVE<br>
 900 West Olympic Boulevard<br>
 Los Angeles, California 90015 USA
 
-More information and an on-site map will be posted closer to the event.
+
+### Lucky Strike
+<img align="right" src="/events/kcsna2021/lucky-strike.jpg" width="30%">
+
+The evening social event will be held at
+<a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">Lucky Strike</a>,
```

Comment:
URL should be: https://www.luckystrikeent.com/locations/los-angeles/ :)

Reply:
fixed 👍